target/linux/octeon/image/Makefile

```diff
@@ -47,4 +47,40 @@ define Device/ubnt_edgerouter-lite
 endef
 TARGET_DEVICES += ubnt_edgerouter-lite
 
+#
+# Itus Network Shield Profiles
+#
+ITUSROUTER_CMDLINE:=console=ttyS0,115200, root=/dev/mmcblk1p2 rootfstype=squashfs,ext4,f2fs rootwait
+define Device/itusrouter
```

Comment:
Please use the currently common vendor_model style here, e.g.
itus_shield-router

I'd also be happy if you could use the board name similar to what's done on modern DTS targets, i.e. itus,shield-router based on the suggested name.